sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessorCheckpoint.cs

```diff
@@ -39,6 +46,12 @@ public class EventProcessorCheckpoint
         ///
         public string PartitionId { get; set; }
 
+        /// <summary>
+        /// The date and time the checkpoint was last modified.
+        /// </summary>
+        ///
+        public DateTimeOffset? LastModified {  get; set; }
```

Comment:
Debating if there should also be a `DateTimeOffset TimeCreated`? Maybe just in the blob checkpoint implementation?